combat-trainer.lic

```diff
@@ -1336,6 +1336,8 @@ class TrainerProcess
 
     @analyze_retry_count = settings.combat_analyze_retry_count
     echo("  @analyze_retry_count: #{@analyze_retry_count}") if $debug_mode_ct
+
+    @combat_training_abilities_target = settings.combat_training_abilities_target
```

Comment:
Can you add an echo for this setting (similar to line 1338)?

Reply:
All set